scripts/devstack.sh

```diff
@@ -70,9 +70,14 @@ stop()  {
   #############################################################################
   # Stops and unmounts a devstack servers.                                                   #
   #############################################################################
-  unmount
-  make stop
-	success "Your devstack stopped successfully."
+  if nc -z "$SSH_AGENT_HOST_NAME" 22 2>/dev/null; then
+    unmount
```

Comment:
Reminder that `unmount` doesn't exist on Linux.

Reply:
Yes, this has been fixed in #49. This is just a call to the function that has the unmount logic: https://github.com/appsembler/sultan/pull/55/files/fcbe0ddc5258b1e7941d5b5394f84dce400f4e4f#diff-c1ffe1cc97964684798d658877a05c0ad34224c06383e72f3e2683db1caf944bR53